packages/sw-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -117,7 +120,16 @@ class RequestWrapper {
     assert.atLeastOne({request});
 
     const cache = await this.getCache();
-    return await cache.match(request, this.matchOptions);
+    const response = await cache.match(request, this.matchOptions);
+
+    // Give the cacheWillMatch callbacks a chance to modify the cached response.
```

Comment:
These are inline comments, not JSDoc. They don't get run through the Markdown processor or included as part of the published documentation, so Markdown syntax seems unneeded.